lib/ansible/modules/packaging/os/portage.py

```diff
@@ -326,13 +331,18 @@ def emerge_packages(module, packages):
         module.fail_json(msg='Use only one of usepkg, usepkgonly')
 
     emerge_flags = {
-        'jobs': '--jobs=',
-        'loadavg': '--load-average ',
+        'jobs': '--jobs',
+        'loadavg': '--load-average',
     }
 
     for flag, arg in emerge_flags.items():
-        if p[flag] is not None:
-            args.append(arg + str(p[flag]))
+        flag_val = p[flag]
+        if not flag_val:
+            if flag_val is not None:
+                args.append(arg)
+            continue
+
+        args.extend((arg, to_native(flag_val)))
```

Comment:
not sure if its just me but i still find it confusing, wouldn't it be cleaner as?:

```
        flag_val = p[flag]
        if flag_val:
           args.extend((arg, to_native(flag_val)))
        elif flag_val is not None:
           args.append(arg)
```

Reply:
I tried to avoid nesting here. Maybe I can reduce it to be one-level.

Reply:
@bcoca It must be cleaner now

Reply:
Thanks for making it clearer and comments. Though I think I prefer bcoca approach. Less code, and does not require comments. I read it as either add the flag and value, or if not none, just add the flag. If neither, it does nothing and onto the next iteration. Does not require any continues. Seems more compact and elegant. But either can work. More what the project prefers, so would defer to bcocas preference.

Reply:
At the point it is 'clear' i'm fine with either way, i am not going to force a style unless the code need de-obfuscation.

Reply:
Ok I will leave it to the contributor. Rather not debate over stylish stuff that is moot. I am good either way, long as it works. Thanks!